Description: Normalise expected cavern output
 Strip the line which contains how long cavern took to run.  This is usually
 "0.00" seconds for the tiny testcases with the timer granularity, but it
 can be enough to report an actual time and cause the test to fail.
Author: Olly Betts <olly@survex.com>
Forwarded: no
Last-Update: 2012-05-16

--- survex-1.2.6.orig/tests/equatenosuchstn.out
+++ survex-1.2.6/tests/equatenosuchstn.out
@@ -24,6 +24,5 @@ East-West range = 9.70m (from entrance.2
 ./equatenosuchstn.svx:7: Reference to station "nosuchsurvey.1" from non-existent survey "nosuchsurvey"
    2 1-nodes.
    1 2-node.
-CPU time used  0.00s
 Done.
 There were 1 warning(s) and 1 error(s) - no output files produced.
--- survex-1.2.6.orig/tests/cavern.tst
+++ survex-1.2.6/tests/cavern.tst
@@ -272,9 +272,9 @@ for file in $TESTS ; do
     if test -f "$file.out" ; then
       # Check output is as expected.
       if test -n "$VERBOSE" ; then
-	sed '1,/^$/d' tmp.out|diff - "$file.out" || exit 1
+	sed '1,/^$/d;/^\(CPU t\|T\)ime used  *[0-9][0-9.]*s$/d' tmp.out|diff - "$file.out" || exit 1
       else
-	sed '1,/^$/d' tmp.out|diff - "$file.out" > /dev/null || exit 1
+	sed '1,/^$/d;/^\(CPU t\|T\)ime used  *[0-9][0-9.]*s$/d' tmp.out|diff - "$file.out" > /dev/null || exit 1
       fi
     fi
     rm -f tmp.*
